### PR TITLE
CCD-965: Remove Temporary Sonar Exclusions ccd-case-document-am-api

### PR DESCRIPTION
**Pull request by JamesCollettCGI**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CCD-965

### Change description ###
CCD-965: Remove Temporary Sonar Exclusions ccd-case-document-am-api

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by JamesCollettCGI** [comment]
**PR build:** https://build.platform.hmcts.net/job/HMCTS_a_to_c/job/ccd-case-document-am-api/view/change-requests/job/PR-570/
**PR sonar:** https://sonarcloud.io/summary/new_code?id=ccd-case-document-am-api&pullRequest=570

File: build.gradle [hunk 1]
```diff
@@ -204,8 +204,6 @@ sonarqube {
     property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/reports/jacoco/test/jacocoTestReport.xml"
     property "sonar.pitest.mode", "reuseReport"
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
-    // FIXME : RDM-11309
-    property "sonar.exclusions",  "**/service/impl/DocumentManagementServiceImpl.java,"
     property "sonar.coverage.exclusions", "**/configuration/*.java,**/advice/exception/*.java,**/apihelper/*.java,**/ErrorConstants.java,**/ClassificationValueValidator.java,**/UpdateDocumentCommand.java,**/UpdateDocumentsCommand.java,**/AuthorisedService.java"
```
Comment (rajaram-nagarajan) [inline build.gradle:207]: @JamesCollettCGI - Can you please evaluate sonar.coverage.exclusions property as well?

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by dinesh1patel — APPROVED** [review]
LGTM

**Pull request merged**